iidm/iidm-tck/src/main/java/com/powsybl/iidm/network/tck/AbstractTapChangerTest.java

```diff
@@ -220,18 +222,7 @@ public void testTapChangerSetterGetterInMultiVariants() {
 
         variantManager.setWorkingVariant("s4");
         // check values cloned by extend
-        assertEquals(1, phaseTapChanger.getTapPosition());
-        assertFalse(phaseTapChanger.isRegulating());
-        assertEquals(1.0, phaseTapChanger.getRegulationValue(), 0.0);
-        assertEquals(1, ratioTapChanger.getTapPosition());
-        assertTrue(ratioTapChanger.isRegulating());
-        assertEquals(10.0, ratioTapChanger.getTargetV(), 0.0);
-        assertEquals(1, ratioTapChangerInLeg2.getTapPosition());
-        assertTrue(ratioTapChangerInLeg2.isRegulating());
-        assertEquals(10.0, ratioTapChangerInLeg2.getTargetV(), 0.0);
-        assertEquals(3, ratioTapChangerInLeg3.getTapPosition());
-        assertFalse(ratioTapChangerInLeg3.isRegulating());
-        assertEquals(11.0, ratioTapChangerInLeg3.getTargetV(), 0.0);
+        assertKnownState(phaseTapChanger, ratioTapChanger, ratioTapChangerInLeg2, ratioTapChangerInLeg3);
```

Comment:
Could you certify this is equivalent?

Reply:
If I did this correctly, it is equivalent (asserKnownState is juste a newly extracted method to avoid code duplication detected by sonar).